cyb_oko/settings.py

```diff
@@ -36,6 +36,7 @@
     'voucher',
     'z',
     'members',
+    'intern',
```

Comment:
Yeah, I see your point about that. Guess `intern` is valid, I don't really have any other idea anyways.